wsclient/src/main/java/org/hpccsystems/ws/client/antlr/EclRecord.g4

```diff
@@ -40,14 +40,14 @@ record_def_inline:
 ;
 
 record_def:
-    REC_SYM (COMMA maxlength)? eclfield_decl SEMI (eclfield_decl SEMI)* END_SYM SEMI
+    REC_SYM (COMMA maxlength)? eclfield_decl SEMI comment? (eclfield_decl SEMI comment?)* END_SYM SEMI
 ;
 defined_record_def :
     TOKEN ASSING_SYM (record_def|record_def_inline)
 ;
 
 exploded_dataset_record_def:
-    REC_SYM (COMMA maxlength)? eclfield_decl SEMI (eclfield_decl SEMI)* END_SYM
+    REC_SYM (COMMA maxlength)? eclfield_decl SEMI comment? (eclfield_decl SEMI comment?)* END_SYM
```

Comment:
They look like they're handled the exact same way in the eclreader code.  Perhaps this is something we should look at in the future